source/PlayerInfo.cpp

```diff
@@ -2016,6 +2018,58 @@ void PlayerInfo::SetTravelDestination(const Planet *planet)
 
 
 
+void PlayerInfo::QueueTeleport(const Planet *teleportPlanet, bool _flagshiponly)
+{
+	teleportDestination = teleportPlanet;
+	flagshiponly = _flagshiponly;
+}
+
+
+
+void PlayerInfo::DoQueuedTeleport()
+{
+	if(teleportDestination && planet)
+	{
+		flagship->SetSystem( teleportDestination->GetSystem());
+		flagship->SetPlanet(teleportDestination);
+		if(!flagshiponly)
+		{
+			for(const shared_ptr<Ship> &ship : ships)
+			{
+				if(!ship->IsParked() && !ship->IsDestroyed() && !ship->IsDisabled() && ship->GetSystem() == system)
```

Comment:
This line has gotten too long
Also its probably best to check if the ship is on the same planet (and thus removing the need to check for the system) rather than if it isnt disabled

Reply:
done, probably

Reply:
Yeah its good, that naturally made the line shorter
It should prob be clarified in the pr description it only affects ship on the same planet as the player

Reply:
done